ansible/inventories/production/hosts

```diff
@@ -0,0 +1,2 @@
+[stubbornjava]
+54.84.142.75
```

Comment:
You should use the ec2 inventory script in the future.
http://docs.ansible.com/ansible/latest/intro_dynamic_inventory.html#example-aws-ec2-external-inventory-script

Reply:
Once we get terraform working I'll do that.